scripts/generate_builtins.py

```diff
@@ -40,110 +42,114 @@ def trimext(file: str) -> str:
     ],
 }
 
-# There's a number of functions we _don't_ support yet, so we exclude emitting these to the file
-unsupported = set([
+# There's a number of functions we only support in the new experimental converter
+experimental = set([
```

Comment:
Is there any way to tell what is and is not supported, or is it just an exhaustive list? Can this comment point to a place in our code to check if the list is correct?


P.S. Python has set literals:
```suggestion
experimental = {
```

Reply:
> Is there any way to tell what is and is not supported, or is it just an exhaustive list? 

In this PR nothing. There's a list of handled functions in the new converter, so this could point to that.

Reply:
I meant "Is there a list anywhere in our code base". Pointing to the list of handled functions would be awesome!

Reply:
> the list of handled functions would be awesome

I mean that's what this script generates :) It grabs every function from TF's docs and unless it's marked "unsupported" here writes it to the builtins.tf test file which the converter then checks can convert to pcl.